src/lib.rs

```diff
@@ -662,7 +662,7 @@ where
         D: Deserializer<'de>,
     {
         deserializer.deserialize_tuple(N, ArrayVisitor::<[T; {N}]>::new())
-            .map(|a|Vector(a))
+            .map(Vector)
```

Comment:
It blows my mind that the name of tuple structs coerce to constructor functions. Wow!!!